pom.xml

```diff
@@ -37,6 +37,17 @@
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+      <dependency>
+          <groupId>org.jetbrains</groupId>
+          <artifactId>annotations</artifactId>
```

Comment:
You do not use it. It is not a good practice to add any  unused dependency (maybe it was inserted by your IDEA)